README.md

```diff
@@ -0,0 +1,51 @@
+# Toronto Community Network
+
+TBA
+
+## Project Management
+We use [Teams](https://github.com/orgs/tomeshnet/teams/toronto-community-network/teams) to manage membership of working groups and [Projects](https://github.com/tomeshnet/toronto-community-network/projects/1) to track task status on a public project board.
+These are our current [working groups](https://github.com/orgs/tomeshnet/teams/toronto-community-network/teams).
```

Comment:
@darkdrgn2k pointed out these aren't visible for non-members. It seems like a github bug? They are made public, so I'll leave it.